applications/community-health-check.md

```diff
@@ -0,0 +1,281 @@
+
+### Problem Solution
+We are offering community health checks that are based on the relationships between people and their sense of belonging and ownership in the community. We do this by creating and implementing a science-based community health framework. This framework rests on years of research on communities and social networks.
+
+Our community health checks combine objective data (interactions between contributors) and subjective data (contributors perceptions about the community). Interactions between contributors are a vital data point: During interactions, members exchange knowledge, develop ideas, and influence each others perspective. Therefore, a community dashboard that measures interaction patterns, compared to solely activity, acknowledges that contributors do not act in isolation. Measuring perception data through pulse survey (tiny surveys with 3-5 questions) gives community leaders a quick and easily administered insights into how contributors feel.
+
+Although our initial focus is community health metrics, the proposed approach sets the foundation for further applications. The combination of SNA and pulse surveys offers unprecedented actionable insights in real-time. Some of the potential applications and insights for the ecosystem are
+
+- Generate baseline metrics for community health / vibez to quantify and better understand the impact a specific event is having on a community and/or sub-groups within the community
+- Identify measurements of decentralization to serve as KPIs or Insights metrics
+- Build funnels to track member onboarding and identify areas for improvement
+- Monitor specific topics like contributor wellbeing, alignment, community experience, etc. in near real-time
+- Attract talent and investment with objective community health metrics instead of proxy metrics like member count or proposal count, or financial metrics such as TVL
+- Help new contributors find context-rich mentors outside of the existing pool of well-known but time-poor candidates
+
+### Rationale 
+
+Social network data and its methodology (Social Network Analysis; SNA) is a structured way to visualize how communications, information, and value creation occur through an organization based on interaction graphs. Although a relatively new decision-making tool for leaders, SNA is gaining popularity over traditional survey tools. SNA has been linked to a wide range of insights, such as improving contributor retention, avoiding member burnout, predicting team performance and community resilience, identifying key contributors, enabling decentralization, and improving coordination.
+
+![Data points for measuring community health](howmeasurecommhealth.jpg)
```

Comment:
```suggestion
![Data points for measuring community health](https://user-images.githubusercontent.com/5393704/224280603-03e8b7a7-4212-4a46-b957-ef06ebe3de5a.png)
```

Ideally, we avoid polluting the `applications` directory with images and reference them from an external URL instead.